pyk/src/pyk/prelude.py

```diff
@@ -89,52 +87,55 @@ def token(x: Union[bool, int, str]) -> KToken:
     raise AssertionError()
 
 
-def intToken(i: int) -> KToken:
+def intToken(i: int) -> KToken:  # noqa: N802
     return KToken(str(i), Sorts.INT)
 
 
-def stringToken(s: str) -> KToken:
+def stringToken(s: str) -> KToken:  # noqa: N802
     return KToken(f'"{s}"', Sorts.STRING)
 
 
-def ltInt(i1, i2):
+def ltInt(i1, i2):  # noqa: N802
     return KApply('_<Int_', i1, i2)
 
 
-def leInt(i1, i2):
+def leInt(i1, i2):  # noqa: N802
     return KApply('_<=Int_', i1, i2)
 
 
 # TODO default sort K can be tightened using basic type inference
-def mlEquals(
-    term1: KInner, term2: KInner, sort1: Union[str, KSort] = Sorts.K, sort2: Union[str, KSort] = Sorts.K
+def mlEquals(  # noqa: N802
+    term1: KInner,
+    term2: KInner,
+    sort1: Union[str, KSort] = Sorts.K,
+    sort2: Union[str, KSort] = Sorts.K,
```

Comment:
I think we should consider renaming these to `sort_args` and `sort_return` or something, instead of `sort1/2`, because people might think it means the sorts of `term1` and `term2`, when it doesn't. Sort1 should be the sort of both arguments, sort2 is the sort that the result is projected into.

Doesn't have to be this PR

Reply:
Done.